### ISSUE TITLE
Replace deprecated date widget in newsfeed

### ISSUE THREAD

**Issue by nholden** [texas-justice-initiative/website-nextjs#303]
@EvaRuthM [bumped into some issues](https://txjusticeinitiative.slack.com/archives/CACFGUQG0/p1587153794002900) adding new items to the "what we're reading" list on the homepage. I suspect it has to do with the [Netlify CMS date widget](https://www.netlifycms.org/docs/widgets/date/), which is deprecated.

Let's replace the deprecated date widget with something else and see if we that addresses the issues with adding new items to the newsfeed.

**Comment by nholden** [comment]
I spent about an hour on this today, and I got frustrated and had to step away. Here's what I learned.

- I tried using the [`datetime` widget](https://www.netlifycms.org/docs/widgets/#datetime) instead of the `date` widget. I kept the `format` as `YYYY-MM-DD` and set `timeFormat` to `false`. Whenever I tried saving a date, the date CMS would save the day before to Markdown (e.g. I select 2020-04-19, the CMS would persist 2020-04-18).
- I tried using the [`string` widget](https://www.netlifycms.org/docs/widgets/#string) with format validation. I think this didn't work because our Markdown parser [automatically converts things that look like dates](https://github.com/hmsk/frontmatter-markdown-loader/issues/134).
